caravel/assets/stylesheets/less/bootswatch.less

```diff
@@ -415,26 +414,29 @@ input[type="checkbox"],
 
 .nav-tabs {
   > li > a,
+  > li > a:hover
   > li > a:focus {
     margin-right: 0;
     background-color: transparent;
-    border: none;
+    border: 1px solid #ccc;
+    margin-right: 1px;
+    border-radius: 10px 10px 0px 0px;
```

Comment:
Yeah I don't like the way they look now, plus they somehow disappear on hover and couldn't get to the bottom of it. Since there's no tabs in master this doesn't matter much at this point, I was thinking we can do another round on tabs.